types/events.go

```diff
@@ -81,12 +82,21 @@ type EventDataRoundState struct {
 	RoundState interface{} `json:"-"`
 }
 
+type EventDataNewRound struct {
+	Height int64  `json:"height"`
+	Round  int    `json:"round"`
+	Step   string `json:"step"`
+	ProposerAddress Address `json:"proposer_address"`
```

Comment:
What about packaging these in a new `ProposerInfo` type containing `Address` and `Index`. Just thinking it might be more future proof if we change stuff about proposers

Reply:
sounds good. should we make it slightly more generic and call it `ValidatorInfo`? For now it would contain a pair of the existing `Validator` object and an `ValidatorIndex int`?

Reply:
kept it simple for now. let me know if we want to expand